app/services/organizations/delete.rb

```diff
@@ -5,7 +5,10 @@ def initialize(org)
     end
 
     def call
+      self.article_ids = org.article_ids
```

Comment:
It would make sense to me to move this to `initialize` as it sets up the class' state.

Reply:
+1 on moving this in the initializer